src/templates/partials/source-file.html

```diff
@@ -54,9 +56,7 @@
     </span>
     <nav>
       {% for author in authors %}
-        <a href="mailto:{{ author.email }}">
-          {{- author.name -}}
-        </a>
+        {%- if show_email_address %}<a href="mailto:{{ author.email }}">{{- author.name -}}</a>{%- else -%}{{- author.name -}}{% endif -%}
```

Comment:
This is not really readable. Please adhere to the surrounding formatting, so move things onto separate lines. There's also no need to put `show_email_address` in a separate variable I think.

Reply:
Roger that, updated.